Readme.md

```diff
@@ -64,6 +64,12 @@ type Options = {
 
     // Any graphql operations containing ignorePragma will be skipped
     ignorePragma?: string,
+
+    // Set to true to mirror gqlgen's behavior of exporting all
+    // nested object types within the response type.
+    // Names are generated by concatenating the attribute names
+    // of the path to the object type, separated by underscores.
+    exportAllObjectTypes: boolean,
```

Comment:
nit: can make this optional like the `splitTypes` boolean?

Reply:
oh right it is